changelog.md

```diff
@@ -1,4 +1,22 @@
- <a name="6.4.0"></a>
+<a name="6.4.1"></a>
+# 6.4.1 (2020-10-02)
+[Full Changelog](https://github.com/GoogleChrome/lighthouse/compare/v6.4.0...v6.4.1)
+
+This is a minor release to fix an issue in the npm package where v6.4.0 was published as ~15GB.
```

Comment:
hmm, I misread the severity of this.

Reply:
yeah sorry if I misled you @connorjclark I was trying to say this probably *wasn't* a showstopper since npm installing *all of our deps* is apparently 15GB so us increasing ~50MB isn't the end of the world, but this fix combined with the many malformed request issue in #11503  would be a nice 6.4.1 anyway if you want to continue :)

Reply:
yea might as well keep going, but I probably won't follow up with a CDT release.

Reply:
it was my fault, I read it quickly this morning then went off to do a bunch of errands, so all day this was in my head and as soon as I got to a computer I didn't check any of my assumptions